explore/src/main/scala/explore/shortcuts.scala

```diff
@@ -23,29 +34,37 @@ type ShortcutCallbacks = PartialFunction[Shortcut, Callback]
 
 given Conversion[PartialFunction[Shortcut, Callback], HotkeysCallback] with
   def apply(p: PartialFunction[Shortcut, Callback]): HotkeysCallback =
-    (e: HotkeysEvent) => p.applyOrElse(Shortcut(e.keys.toList), _ => Callback.empty)
+    (e: HotkeysEvent) =>
+      Shortcut
+        .withModifiers(e)
+        .collectFirst {
+          case s if p.isDefinedAt(s) =>
+            p.applyOrElse(s, _ => Callback.empty)
+        }
+        .headOption
+        .getOrElse(Callback.empty)
 
 val GoToObs         = Shortcut("o")
 val GoToTargets     = Shortcut("t")
 val GoToProposals   = Shortcut("r")
-val GoToConstraints = Shortcut("c")
+val GoToConstraints = Shortcut("n")
 val GoToOverview    = Shortcut("w")
 val GoToSummary     = Shortcut("s")
 
 val Esc           = Shortcut("ESC")
-val ShortcutsHelp = Shortcut("shift+/")
+val ShortcutsHelp = Shortcut("F1")
 
 val Down = Shortcut("j")
 val Up   = Shortcut("k")
 
 val CopyAlt1 = Shortcut("y")
 val CopyAlt2 = Shortcut("ctrl+c")
```

Comment:
I'm not sure how this all works, but there is no `ctrl` in the `withModifiers` method. Is that OK?

Reply:
Right, I should delete these, `meta` should cover `cmd` and `ctrl`